pkg/ebpf/tracer_windows.go

```diff
@@ -2,23 +2,150 @@
 package ebpf
 
+/*
+// https://github.com/DataDog/datadog-windows-filter/blob/master/include/ddfilterapi.h#L29
+typedef struct _stats
+{
+    long Read_calls;		//! number of read calls to the driver
+    long Read_bytes;
+    long Read_calls_outstanding;
+    long Read_calls_completed;
+    long Read_calls_cancelled;
+    long Read_packets_skipped;
+    long Write_calls;	//! number of write calls to the driver
+    long Write_bytes;
+    long Ioctl_calls;	//! number of ioctl calls to the driver
+} STATS;
+
+typedef struct driver_stats
+{
+    STATS		Total;		//! stats since the driver was started
+    STATS		Handle;		//! stats for the file handle in question
+} DRIVER_STATS;
+*/
+import "C"
 import (
+	"expvar"
+	"fmt"
+	"time"
+	"unsafe"
+
 	"github.com/DataDog/datadog-agent/pkg/process/util"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"golang.org/x/sys/windows"
+)
+
+const (
+	DRIVERFILE = `\\.\ddfilter`
+
+	// https://github.com/DataDog/datadog-windows-filter/blob/master/include/ddfilterapi.h
+	DDFILTER_IOCTL_GETSTATS               = 0x801
+	DDFILTER_IOCTL_SIMULATE_COMPLETE_READ = 0x802
+	DDFILTER_IOCTL_SET_FILTER             = 0x803
+
+	// https://docs.microsoft.com/en-us/windows-hardware/drivers/kernel/specifying-device-types
+	NETWORK_DEVICE_TYPE_CTL_CODE = 0x00000012
+
+	// Number of buffers to be used with the IOCompletionPort
+	TOTAL_READ_BUFFERS = 10
 )
 
+var (
+	expvarEndpoints map[string]*expvar.Map
+	expvarTypes     = []string{"driver_total_stats", "driver_handle_stats"}
+)
+
+func init() {
+	expvarEndpoints = make(map[string]*expvar.Map, len(expvarTypes))
+	for _, name := range expvarTypes {
+		expvarEndpoints[name] = expvar.NewMap(name)
+	}
+}
+
 // Tracer struct for tracking network state and connections
 type Tracer struct {
-	config *Config
+	config       *Config
+	driverHandle windows.Handle
 }
 
 // NewTracer returns an initialized tracer struct
 func NewTracer(config *Config) (*Tracer, error) {
-	return &Tracer{}, nil
+	handle, err := openDriverFile(DRIVERFILE)
+	if err != nil {
+		return nil, fmt.Errorf("%s : %s", "Could not create driver handle", err)
+	}
+	tr := &Tracer{
+		driverHandle: handle,
+	}
+	go tr.expvarStats()
+	return tr, nil
+}
+
+func (t *Tracer) expvarStats() {
+	ticker := time.NewTicker(5 * time.Second)
+	// starts running the body immediately instead waiting for the first tick
+	for range ticker.C  {
+		stats, err := t.GetStats()
+		if err != nil {
+			continue
+		}
+
+		for name, stat := range stats {
+			for metric, val := range stat.(map[string]int64) {
+				currVal := &expvar.Int{}
+				currVal.Set(val)
+				expvarEndpoints[name].Set(snakeToCapInitialCamel(metric), currVal)
+			}
+		}
+	}
 }
 
 // Stop function stops running tracer
 func (t *Tracer) Stop() {}
 
+func openDriverFile(path string) (windows.Handle, error) {
+	p, err := windows.UTF16PtrFromString(path)
+	if err != nil {
+		return windows.InvalidHandle, err
+	}
+	log.Debug("Creating Driver File...")
+	r, err := windows.CreateFile(p,
```

Comment:
You're right, and I'm updating this. Especially since taking a second look at this, createFile actually creates handle. 

The one thing of note is that you can't return `nil` as an a handle type, so I'll be passing back something like this: